src/Compilers/CSharp/Portable/Binder/Semantics/Conversions/Conversion.cs

```diff
@@ -513,6 +515,14 @@ public bool IsThrow
             }
         }
 
+        internal bool IsNew
+        {
+            get
+            {
+                return Kind == ConversionKind.ImplicitNew || this.UnderlyingConversions.AsSingleton().Kind == ConversionKind.ImplicitNew;
```

Comment:
>AsSingleton() [](start = 88, length = 13)

`Single()`

Reply:
We don't want to throw in any case, as it turns out AsSingletone might fail for ImmutableArray (calls Count on default value).